code/src/components/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+import { UserName } from './Name'
+import { AgeRadioOption } from './Age'
+import { SelectPlanet } from './SelectPlanet'
+import { BringItem } from './BringItem'
+import { Summary } from './Summery'
+
+
+
+
+export const Form = () => {
+  const [ageGroupSpan, setAgeGroup] = useState('')
```

Comment:
I would consider the naming of the useState here. You set the state to “ageGroupSpan” but only named the set function to “setAgeGroup”. In the spirit of being consistent with naming states, a suggestion is to name the set function to “setAgeGroupSpan”. 🙂